client/src/components/CreatePost/Form/OfferHelp.js

```diff
@@ -3,15 +3,25 @@ import First from "./FirstSection";
 import Second from "./SecondSection";
 import Third from "./ThirdSection";
 import { Footer, Submit } from "components/CreatePost/StyledModal";
+import createPostSettings from "assets/data/createPostSettings";
+
+const { helpTypes } = createPostSettings;
 
```

Comment:
This file was also deleted so probably gonna be a lot of merge conflicts 😨

Reply:
I'm working on fixing the merge conflict rn. Thanks for letting me know the PR that led to these conflicts @idjevm that was super helpful!  if you see anyone making changes to the post/feed please lmk

Reply:
Everything looks good except I believe through the merge resolution the two unused form components were added back - AskHelp.js & OfferHelp.js - I will approve so you can merge right after deleting those two files first:

 client/src/components/CreatePost/Form/AskHelp.js
 client/src/components/CreatePost/Form/OfferHelp.js